frontend/components/wallet/WalletProvider.tsx

```diff
@@ -191,15 +191,24 @@ export const WalletProvider: React.FC<Props> = ({ children }) => {
     };
 
     try {
-      console.log("LISTENING TO REFINE TO ARCHIVE EVENT");
+      console.log("LISTENING TO REFINE TO ARCHIVE AND TRANSFER EVENTS");
       contract.on("RefineToArchive", (sender, value) => {
         console.log("REFINE TO ARCHIVE EVENT", sender, value);
-        setRefineEventCounter((prevCounter) => {
+        setUserActionCounter((prevCounter) => {
+          return prevCounter + 1;
+        });
+      });
+      contract.on("Transfer", (sender, value) => {
+        console.log("TRANSFER EVENT", sender, value);
+        setUserActionCounter((prevCounter) => {
```

Comment:
this is going to duplicate per second the amount of calls we make. Do we really need this?

Reply:
once minting the interface does not update stats so it makes it confusing from user perspective @Ledespana